### PR TITLE
chore: v1.0.0-beta.109

### PR DESCRIPTION
**Pull request by RomanHotsiy**
### Features

- Added rfc7807 problem details rule. (#816)
- Improved error messages by adding `referenced from` information. (https://github.com/Redocly/redocly-cli/pull/835)
- Added a rule for component map names validation. (#815)
- Added a new `--format` option: `summary`. (#860)

### Fixes

- Fixed an issue with multi-line strings in literal mode. (#822)
- Fixed an issue with multi-line Markdown with Windows-style new lines. (#827)
- Fixed a Header object spec to require `content` or `schema`. (#830)
- Fixed a error message for `operation-4xx-response` rule (closes https://github.com/Redocly/redocly-cli/issues/840).
- Fixed an issue with `paths` not being correctly handled by the `join` command. (#778)
- Fixed the `operation-security-defined` rule to check the security on the root and in each operation. (#862)

### Changes

- Renamed 'DefinitionRoot', 'ServerVariableMap', 'PathMap', 'CallbackMap', 'MediaTypeMap', 'ExampleMap', 'EncodingMap', 'HeaderMap', and 'LinkMap' definition node types. (#852)
- Removed the `styleguide` object from the configuration file. (#847)
- Renamed the `operation-security-defined` rule to `security-defined`. (#862)

## Check yourself

- [x] Code is linted
- [x] Tested with redoc/reference-docs/workflows
- [ ] All new/updated code is covered with tests

## Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -4,6 +4,23 @@ tocMaxDepth: 2
...
+- Added a rule for component names validation.
+- Added a new `--format` option: `summary`.
+
+
+### Fixes
+
+- Fixed an issue with multiline strings in literal mode.
+- Fixed an issue with multiline markdown with windows-style new lines.
+- Fixed a Header object spec to require `content` or `schema`.
+- Fixed a error message for `operation-4xx-response` rule.
```
Comment (adamaltman) [inline docs/changelog.md:21]:
```suggestion
### Features

- Added rfc7807 problem details rule.
- Improved error messages by adding `referenced from` information.
- Added a rule for component map names validation.
- Added a new `--format` option: `summary`.


### Fixes

- Fixed an issue with multi-line strings in literal mode.
- Fixed an issue with multi-line Markdown with Windows-style new lines.
- Fixed a Header object spec to require `content` or `schema`.
- Fixed a error message for `operation-4xx-response` rule.
```

**Review by tatomyr — APPROVED** [review]

**Comment by adamaltman** [comment]
@tatomyr I imagine we won't release this until next week. Do you want to include the type rename and styleguide flattening in this too?

**Comment by tatomyr** [comment]
@adamaltman I think we can, but I'd like @RomanHotsiy  to take a look at both as they bring pretty significant changes.

**Comment by RomanHotsiy** [comment]
I reviewed.

**Comment by tatomyr** [comment]
So I'm brushing them up and will include to the release.

**Review by tatomyr — APPROVED** [review]

File: docs/changelog.md [hunk 2]
```diff
@@ -4,6 +4,31 @@ tocMaxDepth: 2
...
+
+
+### Fixes
+
+- Fixed an issue with multi-line strings in literal mode.
+- Fixed an issue with multi-line Markdown with Windows-style new lines.
+- Fixed a Header object spec to require `content` or `schema`.
+- Fixed a error message for `operation-4xx-response` rule.
+- Fixed an issue with `paths` not being correctly handled by the `join` command.
+- Fixed the `operation-security-defined` rule to check the security on the root and in each operation.
```
Comment (adamaltman) [inline docs/changelog.md:24]:
```suggestion
- Fixed and renamed the `operation-security-defined` rule to `security-defined` to check the security in the root and in each operation.
```

File: docs/changelog.md [hunk 3]
```diff
@@ -4,6 +4,31 @@ tocMaxDepth: 2
...
+- Added a rule for component map names validation.
+- Added a new `--format` option: `summary`.
+- Added notification about the new version being available.
+
+
+### Fixes
+
+- Fixed an issue with multi-line strings in literal mode.
+- Fixed an issue with multi-line Markdown with Windows-style new lines.
+- Fixed a Header object spec to require `content` or `schema`.
```
Comment (adamaltman) [inline docs/changelog.md:21]:
```suggestion
- Fixed the Header object type to require `content` or `schema`.
```

File: docs/changelog.md [hunk 4]
```diff
@@ -4,6 +4,31 @@ tocMaxDepth: 2
 
 # Redocly CLI changelog
 
+## 1.0.0-beta.109 (2022-09-08)
+### Features
+
+- Added rfc7807 problem details rule.
+- Improved error messages by adding `referenced from` information.
+- Added a rule for component map names validation.
+- Added a new `--format` option: `summary`.
```
Comment (adamaltman) [inline docs/changelog.md:13]:
```suggestion
- Added a new lint `--format` option: `summary`.
```

**Review by AntonKozachuk — APPROVED** [review]

**Pull request merged**